start_torq_demo.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
+
 # Load the environment
 . ./setenv.sh
```

Comment:
Should this get an absolute path too? Would require updating the path to `torq.q` throughout. Or should it be replaced by `torq.sh`, or a call to that script to start up all the procs?

Reply:
I think probably just leave this one for now, we want to move towards torq.sh for linux especially.  start_torq_demo.sh should really only be used on mac now I would think.  We haven't tested compatibility of torq.sh on mac